ui/src/components/General/CodeEditor/CodeEditor.module.scss

```diff
@@ -7,27 +7,24 @@
 		z-index: var(--bt-z-index-dropdown);
 
 		&__success {
-			display: flex;
-			align-items: center;
-			justify-content: center;
+			display: grid;
+			place-content: center;
 			padding: var(--bt-size-2) var(--bt-size-6);
 			border: var(--bt-size-1) solid green;
 			border-radius: var(--bt-size-5);
 		}
 
 		&__failed {
-			display: flex;
-			align-items: center;
```

Comment:
The placements are not in the center

Now:
 
<img width="511" alt="Screen Shot 2024-01-09 at 8 06 35 pm" src="https://github.com/lifeparticle/binarytree/assets/1612112/9dd33d3e-dc37-4230-9fb5-23f4e885d445">

Before:
<img width="426" alt="Screen Shot 2024-01-09 at 8 06 50 pm" src="https://github.com/lifeparticle/binarytree/assets/1612112/aed4d4a4-a046-423f-be89-352c73fc9527">